### PR TITLE
[3303][15.0][MIG] hr_attendance_rest_time: Migration to 15.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[3303](https://www.quartile.co/web#id=3303&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: hr_attendance_rest_time/models/hr_attendance.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    @api.onchange("employee_id", "check_in", "check_out")
+    def _onchange_manual_update(self):
+        self.manual_update = True
+
+    @api.onchange("employee_id")
+    def onchange_employee(self):
+        employee = self.employee_id
+        self.rest_time = employee.rest_time_standard if employee else False
+        self.work_location_id = employee.work_location_id.id if employee else False
```
Comment (yostashiro) [inline hr_attendance_rest_time/models/hr_attendance.py]:
```suggestion
        self.work_location_id = employee.work_location_id if employee else False
```

File: hr_attendance_rest_time/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+The module does the following:
+
+- Adds 'Standard Rest Time (minutes)' field to Employee > Workplace Information.
+- When attendance checkout is pressed, 'Standard Rest Time (minutes)' is auto-proposed.
+- Adds the following fields to Attendance:
+  - Category, Rest Time, Manual Update Flag, Reason for Manual Update
```
Comment (yostashiro) [inline hr_attendance_rest_time/readme/DESCRIPTION.rst:7]:
```suggestion

  - Category, Rest Time, Manual Update Flag, Reason for Manual Update
```

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Review by yostashiro — APPROVED** [review]

**Review by rinaldifirdaus — APPROVED** [review]
Test the functions of this module. works as expected!
- rest time proposed on attendance
- manual update flag and reason displayed on the manual creation of attendance form
image
image

**Comment by kanda999** [comment]
The PR is used by rmm-custom: https://github.com/qrtl/rmm-custom/pull/80

**Pull request closed without merging**